src/org/traccar/geocoder/JsonGeocoder.java

```diff
@@ -112,6 +145,44 @@ public void failed(Throwable throwable) {
         return null;
     }
 
+    @Override
+    public double getSpeedLimit(
```

Comment:
Because we have a second cache for speeds and I couldn't find a good way to return the speed from the original function without changing the type of the callback (original returns a string), which involves changing all of the files that call it.